jobs/BnRStatusPollerJob.js

```diff
@@ -92,26 +92,55 @@ class BnRStatusPollerJob extends BaseJob {
               }
             });
           })
-          .then(() => eventmesh.apiServerClient.getResource({
-            resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BACKUP,
-            resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
-            resourceId: backup_guid
+          .then(() => lockManager.lock(instance_guid, {
```

Comment:
is this for putting lock for long running backups from older code? is this required? SF2.0 for backup is already in 6A, so this might not be required now right?